5.6/rootfs/etc/supervisor/conf.d/php-fpm.conf

```diff
@@ -1,2 +1,12 @@
 [program:php-fpm]
 command=/usr/sbin/php5-fpm --nodaemonize
+stdout_logfile=/dev/stdout
```

Comment:
is it not supposed to be : /tmp/ instead of /dev/ ?

Reply:
No